spec/unit/multi_select_spec.rb

```diff
@@ -905,4 +909,48 @@ def exit_message(prompt, choices)
       expect(prompt.output.string).to eq(expected_output)
     end
   end
+
+  context "with :min and :max" do
+    let(:choices) { %w[A B C] }
+    before do
+      prompt.on(:keypress) do |e|
+        prompt.trigger(:keydown) if e.value == "j"
+      end
+    end
+    context "with min <= max" do
+      let(:min) { 2 }
+      let(:max) { 2 }
```

Comment:
Please remove nested context and inline all the `let` inside the test body as well as the `before`. I'd also use the min and max directly as integers in the `prompt.multi_select` call.